crates/bevy_input_focus/src/directional_navigation.rs

```diff
@@ -173,7 +173,18 @@ impl DirectionalNavigationMap {
         self.add_edge(b, a, direction.opposite());
     }
 
-    /// Add symmetrical edges between all entities in the provided slice, looping back to the first entity at the end.
+    /// Add symettrical edges between each consecutive pair of entities in the provided slice.
+    ///
+    /// Unlike [`add_looping_edges`](Self::add_looping_edges), this method does not loop back to the first entity.
+    pub fn add_edges(&mut self, entities: &[Entity], direction: CompassOctant) {
+        for i in 0..entities.len() - 1 {
+            let a = entities[i];
+            let b = entities[i + 1];
+            self.add_symmetrical_edge(a, b, direction);
+        }
+    }
```

Comment:
Rewrote it for fun, feel free to ignore
```suggestion
    pub fn add_edges(&mut self, entities: &[Entity], direction: CompassOctant) {
        for &[a, b] in entities.windows(2) {
            self.add_symmetrical_edge(a, b, direction);
        }
    }
```